packages/tldraw/package.json

```diff
@@ -74,7 +76,8 @@
     "lask": "^0.0.29",
     "mobx": "^6.3.8",
     "react": ">=16.8",
-    "react-dom": "^16.8 || ^17.0"
+    "react-dom": "^16.8 || ^17.0",
```

Comment:
Any planning to upgrade to 18 :)

Reply:
We'll have to look at this too! I haven't looked into it, so I can't say for sure whether tldraw will work correctly on React 18.

Reply:
I sure hope so though.